general/update_fars/05_copy_fars_from_tdg_server.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+
+# Export env vars
+export $(grep -v '^#' rds_conn_vars.env | xargs)
+
+ogr2ogr \
```

Comment:
Is this script copything things from our server to the SSPF server just meant for testing, or is it meant to be how its going to be done going forward?